packages/victory-core/src/index.d.ts

```diff
@@ -218,13 +218,23 @@ export class VictoryLabel extends React.Component<VictoryLabelProps, any> {}
 // #region Victory Container
 
 export interface VictoryContainerProps {
+  children?: React.ReactElement | React.ReactElement[];
```

Comment:
One thing I ran into is if you have a child iterator (i.e. if you use map()), React.ReactElement will fail to validate, but React.ReactNode works